app/forms/find/university_degree_status_form.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Find
+  class UniversityDegreeStatusForm
+    include ActiveModel::Model
+    attr_accessor :university_degree_status
+
+    validates :university_degree_status, presence: true
```

Comment:
@kelliedesigner wdyt about the width that Avin raised above?

Reply:
> Tabbing on the "Do you have a university degree?" page skips the second "No, I do not have a degree" radio.

It is because Tabbing should focus on the first radio button in the group. Arrow keys should be used to switch between the options within that group.  When the user presses the Tab key, the focus should move to the entire group of radio buttons. The user can only tab to the group as a whole, not each individual radio button.

Reply:
> <img alt="image" width="844" src="https://private-user-images.githubusercontent.com/50492247/356336591-10025050-53d1-45aa-b6a7-688b88d1bb0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyMTMyOTEsIm5iZiI6MTcyMzIxMjk5MSwicGF0aCI6Ii81MDQ5MjI0Ny8zNTYzMzY1OTEtMTAwMjUwNTAtNTNkMS00NWFhLWI2YTctNjg4Yjg4ZDFiYjBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA5VDE0MTYzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyMjc1YTUxZmI0ZTQyNjZkZjIyZjE3MDI3Yjc2OGEwZTQ1NGI4NzM3OWU0ZDg0OWQ4NjgyMDQ5NjRmNzAxNDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WlTgc4831fFfd4OYJN4rhRczFniezZyVcgYugyVY5rE">
> Just to check, those two banners are different widths, is that what we want?

I think this is fine, as the notification banner is not persistent anyway.

Reply:
> Tabbing on the "Do you have a university degree?" page skips the second "No, I do not have a degree" radio.
> 
> <img alt="image" width="604" src="https://private-user-images.githubusercontent.com/50492247/356336945-e0f5bd9c-dfaf-4743-a7bf-1d06f696521c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyMTMyOTEsIm5iZiI6MTcyMzIxMjk5MSwicGF0aCI6Ii81MDQ5MjI0Ny8zNTYzMzY5NDUtZTBmNWJkOWMtZGZhZi00NzQzLWE3YmYtMWQwNmY2OTY1MjFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA5VDE0MTYzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwY2RiYTJjMTcyYzFiODY2ZTFhZDU0ZTg5ZWZmYjU5YjIxMGMxN2FiZjhiMzM5NWJhZjc3MzUwMmMyNTg5NmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.67o8LAiNAzR3zHIblhN0pXLTpT-HkFbOaRFESR5HQpk">

This is the correct behaviour when navigating with a keyboard, tabbing should move you to the first input of the group, and to toggle between inputs you use the directional arrows.